### DISCUSSION TITLE
panel always wakes up to "home" when using screensaver

### DISCUSSION THREAD

**Discussion by Protoncek** [Blackymas/NSPanel_HA_Blueprint#2053] [General]
Hello!
I'll rather ask here, i don't wanna open a bug report since i'm not sure if it is a bug...
When i set "wake-up-page" in esphome's settings to any other page (but home) and i have screensaver enabled, then touching the screen always wakes up to the home page. Only if i disable screensaver it wakes up to selected wake-up-page. Is that expected or a bug?

**Comment by edwardtfn** [comment]
It's a bug. 😩

**Comment by Protoncek** [comment]
Aha… i discovered it “by mistake”, when i tested all possibilities…
Congrats for your work, it’s really great!
